lib/gman_client/api/health_check.rb

```diff
@@ -0,0 +1,16 @@
+module GmanClient
+  module Api
+    # HealthCheck API
+    module HealthCheck
+      # Retrieve a health check
+      def health_check
+        response = request.api.v1.health_check.get
+
+        unless response.to_h[:health_check] == 'Passed'
```

Comment:
Hmm. I was thinking an exception as I want an alert raised to rollbar but maybe Grotto-agent should raise that exception?

Thoughts? I'm good either way.